src/citrine/resources/data_concepts.py

```diff
@@ -1,6 +1,6 @@
 """Top-level class for all data concepts objects and collections thereof."""
-from abc import abstractmethod, ABC
 import re
+from abc import abstractmethod, ABC
```

Comment:
Because most resources also recommend grouping `import ...` and `from ... import ...` together (e.g. [Reddit's style guide](https://github.com/reddit-archive/reddit/wiki/PythonImportGuidelines)), and it's how do it in most places in CP already.